src/test/java/mops/ArchitectureRulesTest.java

```diff
@@ -0,0 +1,78 @@
+package mops;
+
+import com.tngtech.archunit.core.domain.JavaClass;
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchRule;
+import mops.utils.AggregateRoot;
+import org.junit.jupiter.api.Test;
+
+import java.util.Set;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+import static com.tngtech.archunit.library.Architectures.layeredArchitecture;
+import static com.tngtech.archunit.library.dependencies.SlicesRuleDefinition.slices;
+import static mops.HaveExactlyOneAggregateRoot.HAVE_EXACTLY_ONE_AGGREGATE_ROOT;
+
+@SuppressWarnings("checkstyle:MissingJavadocMethod")
+public class ArchitectureRulesTest {
+
+    private static final String mopsPresentation = "mops.presentation";
+    private static final String mopsBusinesslogic = "mops.businesslogic";
+    private static final String mopsPersistence = "mops.persistence";
+    private final JavaClasses javaClasses = new ClassFileImporter().importPackagesOf(Material1Application.class);
+
+    /**
+     * * This test looks out for public classes that aren't annotated
```

Comment:
something is wrong the formatting of this javadoc.
always use ctrl-alt-L and ctrl-alt-O